core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -52,6 +53,30 @@
   private final boolean mClusterDefaultsLoaded;
 
   /**
+   * WARNING: This API is unstable and may be changed in a future minor release.
+   *
+   * Users should use this API to obtain a configuration for modification before passing to a
+   * FileSystem constructor. The default configuration contains all default configuration params
+   * and configuration properties modified in the alluxio-site.properties file.
+   *
+   * Example usage:
+   *
+   * InstancedConfiguration conf = InstancedConfiguration.defaults();
+   * conf.set(...);
+   * FileSystem fs = FileSystem.Factory.create(conf);
+   *
+   * @return an instanced configuration preset with defaults
+   */
+  public static InstancedConfiguration defaults() {
+    return new InstancedConfiguration(ConfigurationUtils.defaults());
+  }
+
+  /**
+   * WARNING: This API is not intended to be used outside of internal Alluxio code and may be
```

Comment:
In the javadoc typically the first sentence is taken as the preview/short description of what a method does or constructor does. In this case it will always be shown as `WARNING: This API is not intended to be used outside of internal Alluxio code and may be changed or removed in a future minor release.` I would prefer the first sentence for these to say something like `Creates a new instance of {@link InstancedConfiguration}`

Reply:
Isn't that pretty good?